### PR TITLE
UI update and bug fixes

### PR DESCRIPTION
**Pull request by miranmirza**
Updated the login screen to now use a real “sign in with google” button.
Fixed a bug with admins not being able to grant or revoke screener
access.
Separated the CSS from the HTML (in a more elegant way than last time)

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -10,13 +10,13 @@ def show
 
   def revoke_access
     user = User.find(params[:id])
-    user.revoke!
+    user.revoke_screener_access
```
Comment (EmilyMB) [inline app/controllers/users_controller.rb:13]: Good catch!

File: app/assets/stylesheets/common.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
+.exmna-nav {
```
Comment (EmilyMB) [inline app/assets/stylesheets/common.css:1]: :heart: :heart: :heart: Pulling out all the common styles like this @miranmirza

**Comment by EmilyMB** [comment]
The changes look good, but it looks like this is a new bug:
screen shot 2015-08-05 at 3 14 41 pm

**Comment by miranmirza** [comment]
Good catch @EmilyMB. That would be a new bug, I'll fix it now

**Comment by miranmirza** [comment]
I just pushed out a fix that that.

**Comment by EmilyMB** [comment]
@miranmirza LGTM :+1:

**Pull request merged**